pages/linux/pacman-sync.md

```diff
@@ -0,0 +1,36 @@
+# pacman sync
+
+> Arch Linux package manager utility.
+> More information: <https://man.archlinux.org/man/pacman.8>.
+
+- Install a new package:
+
+`pacman --sync {{package_name}}`
+
+- Synchronize and update all packages (add `--downloadonly` for download but do not update anything):
+
+`pacman --sync --refresh --sysupgrade`
+
+- Search the package database for a regular expression or keyword:
+
+`pacman --sync --search "{{search_pattern}}"`
+
+- Display information for a package:
+
+`pacman --sync --info {{package_name}}`
+
+- Install a package as non-explicitly installed:
+
+`pacman --sync --asdeps {{package_name}}`
+
+- Overwrite conflicting files during an update:
```

Comment:
There is no system update like `do-release-upgrade`, some arch-based OS's like `Manjaro` try to imitate that behavior.
Sometimes the user needs to "force" replace a file during a package update, the most common scenario is "change of permissions" and the Arch Linux announcements are used for this news.

Reply:
Okay, thank you.  How about changing it to 'package update', because new users might think 'system update' (like I did)
```suggestion
- Overwrite conflicting files during a package update:
```
What do you think?